src/IECoreHoudini/plugin/Plugin.cpp

```diff
@@ -98,36 +98,60 @@ void newSopOperator(OP_OperatorTable *table)
 {
 	OP_Operator *opHolder = new OP_Operator(
 		"ieOpHolder", "Cortex Op",
-		SOP_OpHolder::create, SOP_ParameterisedHolder::parameters, 0, 4,
-		SOP_ParameterisedHolder::variables, OP_FLAG_GENERATOR
+		SOP_OpHolder::create,
+		SOP_ParameterisedHolder::parameters,
+#if UT_MAJOR_VERSION_INT >= 16
+		SOP_OpHolder::theChildTableName,  
+#endif
+		0, 4, 0, 
```

Comment:
Seem to have dropped `SOP_ParameterisedHolder::variables` for a 0. Was that intentional, and if so, why?